logstash-core-event-java/spec/event_spec.rb

```diff
@@ -135,4 +135,54 @@
       expect(event["tags"]).to eq(["foo"])
     end
   end
+
+  module DummyLogger
+    def self.warn(message)
+      # do nothing
+    end
+  end
+
+  context "logger" do
+
+    let(:logger) { double("Logger") }
+    after(:each) {  LogStash::Event.logger = nil }
+
+    it "should set logger using a module" do
+      LogStash::Event.logger = DummyLogger
```

Comment:
I don't understand what is the purpose of this test if the next one uses a rspec mock?